src/nft/components/profile/view/ProfileAccountDetails.tsx

```diff
@@ -0,0 +1,40 @@
+import { Trans } from '@lingui/macro'
+import { useWeb3React } from '@web3-react/core'
+import Identicon from 'components/Identicon'
+import { MouseoverTooltip } from 'components/Tooltip'
+import useCopyClipboard from 'hooks/useCopyClipboard'
+import { Box } from 'nft/components/Box'
+import { Row } from 'nft/components/Flex'
+import { caption, headlineLarge, lightGrayOverlayOnHover } from 'nft/css/common.css'
+import { useCallback } from 'react'
+import { Copy } from 'react-feather'
+import { shortenAddress } from 'utils'
+
+export const ProfileAccountDetails = () => {
+  const { account, ENSName } = useWeb3React()
+  const [isCopied, setCopied] = useCopyClipboard()
+  const copy = useCallback(() => {
+    setCopied(account || '')
+  }, [account, setCopied])
+
+  return (
+    <Row className={headlineLarge} marginBottom="48" gap="12">
+      <Identicon size={44} />
+      <Box textOverflow="ellipsis" overflow="hidden">
+        {ENSName || shortenAddress(account ?? '')}
+      </Box>
+      <MouseoverTooltip
+        text={
+          <Box className={caption} color="textPrimary">
+            {isCopied ? <Trans>Copied!</Trans> : <Trans>Copy</Trans>}
+          </Box>
+        }
+        placement="right"
+      >
+        <Box paddingX="12" borderRadius="12" cursor="pointer" className={lightGrayOverlayOnHover} onClick={copy}>
+          <Copy strokeWidth={1.5} size={20} />{' '}
```

Comment:
is this necessary `{' '}`?

Reply:
This is auto-added by the ts linter, you can find it throughout the app